templates/pkg/resource/sdk.go.tpl

```diff
@@ -224,7 +224,7 @@ func (rm *resourceManager) updateConditions (
 			errorMessage = err.Error()
 		} else {
 			awsErr, _ := ackerr.AWSError(err)
-			errorMessage = awsErr.Message()
+			errorMessage = awsErr.Code() + ": " + awsErr.Message()
```

Comment:
@surajkota you can just call [`awsErr.Error()`](https://github.com/aws/aws-sdk-go/blob/2ab2a7fda76a11d2df0a0d15e554de2139672d31/aws/awserr/types.go#L65) above, which automatically [produces the code and message](https://github.com/aws/aws-sdk-go/blob/2ab2a7fda76a11d2df0a0d15e554de2139672d31/aws/awserr/types.go#L13).

Reply:
will do

Reply:
done